chef_master/source/ctl_automate_server.rst

```diff
@@ -334,6 +334,45 @@ This subcommand has the following syntax:
 
    $ automate-ctl delete-user ENT_NAME john_smith
 
+.. _delete-runner:
+
+delete-runner
+=====================================================
+The ``delete-runner`` subcommand deletes a remote node configured as a job runner, which was used by Chef Automate to run phase jobs. For more information on runners, please see the `Runners documentation </runners.html>`_.
```

Comment:
Note: We'll need version annotation of whichever stable version of Automate this will go out in when you get that version number.